### ISSUE TITLE
Server monitoring for Live VPS

### ISSUE THREAD

**Issue by pacharanero** [rcpch/rcpch-audit-engine#666]
At the DevOps/Infra meeting this week we discussed that we would need some server monitoring for the Live VPS instance on Azure. Suitable monitoring may be achievable using built in Azure features, and would enable:

* general monitoring of usage of the VPS resources - RAM, CPU, network I/O, and SSD space
* alerting for excess usage which might affect system performance for end-users
* detection of patterns of usage - eg spikes in usage which might help us anticipate future user demand better
* optimisation of the cost of the VPS - scaling down if the VPS resources are not fully required to run the E12 application performantly for our userbase - scaling up as necessary in a similar fashion.

It was agreed @ollyrice would review and present available options and costs.

**Comment by ollyrice** [comment]
It looks as if default monitoring was previously set up for the live and staging servers. I've deleted the staging alerts keeping the live ones active. 

- Emails from the informal alerts are being sent to the shared mailbox incubator@rcpch.ac.uk and resource owners.

- I've adjusted the "VM availablility" alert, changed from informal to critical and have added a push notification to my phone's Azure app. Other options such as SMS can be added if needed.

- I have also enabled Insights monitoring to get more advanced reporting of VM usage.

- Costs are negligible so have enabled these alerts.
